### ISSUE TITLE
`metal plans` does not support `--include`, `--exclude`, or `--filter`

### ISSUE THREAD

**Issue by ctreatma** [StackPointCloud/packet-cli#292]
### What happened?

Some `metal` subcommands were recently updated to use the `metal-go` SDK instead of `packngo`.  As a result of that migration, the updated commands lost support for `--include`, `--exclude`, and `--filter` command-line flags.

I ran the same command with version `0.12.0` and with version `0.15.0` and observed that, when using `0.12.0`, I see metro properties in the JSON output, but when using `0.15.0`, the metro properties are not included.

```sh
# With `metal-cli` 0.12.0
% metal plans get -o json --filter slug=m3.small.x86 --include available_in
[
  {
    "slug": "m3.small.x86",
    ...
    "available_in": [
      {
        "id": "cbb790c1-812f-462a-8597-7229f35dc995",
        "name": "Amsterdam",
        "code": "am6",
        "features": [
          "baremetal",
          "layer_2",
          "backend_transfer",
          "global_ipv4",
          "ibx"
        ],
        "address": {
          "id": "3f0f7df5-3827-45c9-8b31-ebb5a443de61",
          "address": "Duivendrechtsekade 80a",
          "city": "Amsterdam",
          "zip_code": "1096",
          "country": "NL",
          "coordinates": {
            "latitude": "52.337915",
            "longitude": "4.933845"
          }
        },
        "metro": {
          "id": "108b2cfb-246b-45e3-885a-bf3e82fce1a0",
          "name": "Amsterdam",
          "code": "am",
          "country": "NL"
        }
      }
    ],
    ...
  }
]

# With `metal-cli` 0.15.0
% metal plans get -o json --filter slug=m3.small.x86 --include available_in
[
  {
    "available_in": [],
    ...,
    "slug": "a3.large.opt-s4a1",
    ...
  }
]
``` 

### How can we reproduce it?

Run the `metal plans get` command with 0.15.0 using any of `--include`, `--exclude`, or `--filter`.

CLI version (`metal --version`): 0.15.0

**Comment by codinja1188** [comment]
We don't have a filter option enabled in the API spec (--filter slug=m3.small.x86)

**Comment by ctreatma** [comment]
Closed by #294
